UI/DesignWnd.cpp

```diff
@@ -414,11 +414,11 @@ namespace {
     void DeleteFromDisplayedDesigns(const int design_id) {
         auto& manager = GetDisplayedDesignsManager();
 
-        //const auto empire_id = HumanClientApp::GetApp()->EmpireID();
-        //const auto maybe_obsolete = manager.IsObsolete(design_id);
-        //if (maybe_obsolete && !*maybe_obsolete)
-        //    HumanClientApp::GetApp()->Orders().IssueOrder(
-        //        std::make_shared<ShipDesignOrder>(empire_id, design_id, true));
+        const auto empire_id = HumanClientApp::GetApp()->EmpireID();
+        const auto maybe_obsolete = manager.IsObsolete(design_id);
+        if (maybe_obsolete && !*maybe_obsolete)
```

Comment:
Please add a comment explaining this... ie. verify that design with requested id is in the manager and that it is not marked obsolete.

Reply:
Any why is a design being obsolete relevant?